data/shubhamds24

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Shubham Das",
+  "type": "personal",
+  "boi": "Travelling is good for health"
```

Comment:
The please check the spelling for "bio".
Also the JSON is invalid for the same line. It should have a trailing comma.

Here is how it should be 👇
```json
"bio": "Travelling is good for health",
```